pkg/utils/devlink.go

```diff
@@ -0,0 +1,79 @@
+package utils
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/vishvananda/netlink"
+)
+
+// NetlinkFunction is a type of function used by netlink to get data
+type NetlinkFunction func(string, string) (map[string]string, error)
+
+const (
+	// fwAppNameKey is a key to extract DDP name
+	fwAppNameKey = "fw.app.name"
+	pciBus       = "pci"
+)
+
+var (
+	netlinkDevlinkGetDeviceInfoByNameAsMap = netlink.DevlinkGetDeviceInfoByNameAsMap
+)
+
+// IsDevlinkDDPSupportedByPCIDevice checks if DDP of PCI device can be acquired with devlink info command
+func IsDevlinkDDPSupportedByPCIDevice(device string) bool {
```

Comment:
what is the different between the two functions?

Reply:
I have removed ``IsDevlinkDDPSupportedByPCIDevice`` and used ``IsDevlinkDDPSupportedByDevice`` throughout instead